### PR TITLE
Fix some highlight box classes that went wrong in porting and remove …

### PR DESCRIPTION
**Pull request by BjornTwachtmann**
…some superflous grid__inner

can test on /programmes/b071vl2l (related programmes)
/programmes/profiles/z5gWDtzYM5nmgZD7zfgDFy/test-profile (promo and gallery grids)

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Utilities/_highlight-box.scss [hunk 1]
```diff
@@ -122,6 +122,10 @@
         margin-bottom: $wide-spacing;
     }
 
+    .highlight-box-wrapper--grid {
+        margin-bottom: -$wide-spacing !important;
```
Comment (Bobkidbob) [inline src/Ds2013/Presenters/Utilities/_highlight-box.scss]: Is there not a way to do this without `!important`?

Comment (BjornTwachtmann) [reply src/Ds2013/Presenters/Utilities/_highlight-box.scss]: Nope, sorry :(

Comment (BjornTwachtmann) [reply src/Ds2013/Presenters/Utilities/_highlight-box.scss]: It's already using !important for this (see definition above). So it's not a new thing

**Pull request merged**